db/migrations/01_create_chitter_table.sql

```diff
@@ -1 +1,3 @@
 CREATE TABLE peeps(id SERIAL PRIMARY KEY, message VARCHAR(60));
+
+ALTER TABLE peeps ADD date BIGINT;
```

Comment:
Postgres has a more natural datatype for representing dates and times (https://www.postgresql.org/docs/9.1/datatype-datetime.html) and Ruby has a [DateTime class](https://ruby-doc.org/stdlib-2.6.1/libdoc/date/rdoc/DateTime.html) which is a bit easier to work with for this purpose than Timestamp.

This would also allow you to simplify your date-conversion code in the Peep class :)